templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -952,6 +868,158 @@ Resources:
       SumoAccessKey: !Ref SumoAccessKey
       SumoDeployment: !Ref SumoDeployment
 
+  OrganizationsRule:
+    Condition: create_config_recorder_role  
+    Type: AWS::Events::Rule
+    Properties:
+      Name: !Sub ${ParentStackName}-org-update
+      Description: Config Trigger on Organizations update
+      EventPattern:
+        source:
+          - aws.organizations
+        detail-type:
+          - AWS API Call via CloudTrail
+        detail:
+          eventSource:
+            - organizations.amazonaws.com
+          eventName:
+            - AcceptHandshake
+            - CreateAccountResult
```

Comment:
Is accept handshake occurs when some member account accepts the invite ?
Also should we include MoveAccount event as well ?

Reply:
the MoveAccount, Moves an account from its current source parent root or organizational unit (OU) to the specified destination parent root or OU. it's not necessary

Reply:
yes if user moves the account out of the ou then we should disable the config and rest of the AWS services right?

Reply:
MoveAccount It supports in Organization. so we should not disable the config and rest of the AWS services